manifests/params.pp

```diff
@@ -271,4 +271,16 @@
   if $init_style == undef {
     fail('Unsupported OS')
   }
+
+  # Autodetect whether we should use camptocamp/systemd
+  $metadata = load_module_metadata('systemd', true)
+  if empty($metadata) {
```

Comment:
`load_module_metadata` didn't quite do what I expected here.  It only returns empty metadata if the metadata didn't exist.  If the module `systemd` didn't exist at all, `load_module_metadata` throws a parse error. :(